docs/tutorials/basic-skill.md

```diff
@@ -82,21 +82,21 @@ class MySkill(Skill):
         await message.respond('Good thanks! My load average is 0.2, 0.1, 0.1.')
 ```
 
-Our skill is done and opsdroid will be able to answer like in the [Opsdroid main page](https://opsdroid.github.io). The next thing that is left to do is to add the skill to the opsdroid configuration file.
+Our skill is done and opsdroid will be able to respond like in the [Opsdroid main page](https://opsdroid.github.io). The final thing  left to do is to add the skill to the opsdroid configuration file.
```

Comment:
```suggestion
Our skill is done and opsdroid will be able to respond like in the [Opsdroid main page](https://opsdroid.github.io). The final thing left to do is to add the skill to the opsdroid configuration file.
```

Reply:
Thanks for that, I removed the extra space :)